docs/quickstart/deploy-stack.md

```diff
@@ -0,0 +1,187 @@
+UUID:            47d8679066a6
+Status:          RUNNING
+Creation Time:   Wed, 10 Apr 2024 13:58:13 CEST
+Flags:           
+
+========================================= Files Artifacts =========================================
+UUID   Name
+```
+
+### Check port mapping
+
+To see the port mapping within the `cdk-v1` enclave for the `zkevm-node-rpc` service and the
+`trusted-rpc` port, run the following command:
+
+```sh
+kurtosis port print cdk-v1 zkevm-node-rpc-001 http-rpc
+```
+
+You should see output that looks something like this (but won't necessarily be the same):
+
+```sh
+http://127.0.0.1:65240
+```
+
+### Set an environment variable
+
+Let's map the output from the previous step to an environment variable that we can use throughout these instructions.
+
+```sh
+export ETH_RPC_URL="$(kurtosis port print cdk-v1 zkevm-node-rpc-001 http-rpc)"
+```
+
+### Test cast commands
+
+This is the same environment variable that `cast` uses, so you should now be able to run the following command:
+
+```sh
+cast block-number
+```
+
+You should see something like this:
+
+```sh
+890
+```
+
+### Pre-funded account
+
+By default, the CDK is configured in test mode, and this means there is some pre-funded ETH in the admin account that has address: `0xE34aaF64b29273B7D567FCFc40544c014EEe9970`.
+
+Check the balance with the following: 
+
+```sh
+cast balance --ether 0xE34aaF64b29273B7D567FCFc40544c014EEe9970
+```
+
+You should see something like this:
+
+```txt
+100000.000000000000000000
+```
+
+### Send transaction with cast
+
+```sh
+cast send --legacy --private-key 0x12d7de8621a77640c9241b2595ba78ce443d05e94090365ab3bb5e19df82c625 --value 0.01ether 0x0000000000000000000000000000000000000000
+```
+
+You should see something like this as output:
+
+```sh
+blockHash               0xc467523f297a9c0b859bedc8feaf44c3e7462de56f7d7899b2039d9a3cfc421d
+blockNumber             70
+contractAddress         
+cumulativeGasUsed       21000
+effectiveGasPrice       1000000000
+from                    0xE34aaF64b29273B7D567FCFc40544c014EEe9970
+gasUsed                 21000
+logs                    []
+logsBloom               0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
+root                    
+status                  1
+transactionHash         0xa58b3383c1b1f53369723fdc537c88daa03585cb6a89aa49ebd493953d519fdf
+transactionIndex        0
+type                    0
+to                      0x0000000000000000000000000000000000000000
+```
+
+### Send transactions with `polygon-cli` -> ISSUE HERE
```

Comment:
What issues did you get? :)

Reply:
Ah, yes, on my Polygon Mac this didn't work and I believe it's likely due to the lack of root user... so when I tried it on Ubuntu externally it was OK. I'll remove this here, it really was a note to self. The errors on Mac were variants of ```Error: unknown command "700" for "polycli loadtest"```